packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2982,6 +2983,8 @@ export const DEFAULT_COMPONENT_ICONS: {
     NotificationBarBreakoutRoomJoined: React_2.JSX.Element;
     NotificationBarBreakoutRoomClosingSoon: React_2.JSX.Element;
     NotificationBarBreakoutRoomClosed: React_2.JSX.Element;
+    NotificationBarTranscriptionError: React_2.JSX.Element;
+    NotificationBarTranscriptionStartedLocally: React_2.JSX.Element;
```

Comment:
Locally sort of implies the transcription is local (i.e. either local to you and not remote participants, or that the AI transcription happens locally and not in the cloud)

Consider `ByYou` instead?

```suggestion
NotificationBarTranscriptionStartedByYou
```